bcipy/gui/params_form.py

```diff
@@ -311,17 +321,17 @@ def __init__(self, parent, title='BCI Parameters',
         self.form = Form(self, json_file)
         vbox.Add(static_text_control(self, label=title, size=20),
                  0, wx.TOP | wx.ALIGN_CENTER_HORIZONTAL, border=5)
-        vbox.AddSpacer(10)
 
         loading_box = wx.BoxSizer(wx.VERTICAL)
         loading_box.Add(static_text_control(self, label=f'Editing: {json_file}',
                                             size=14))
+        loading_box.AddSpacer(7)
         self.loaded_from = static_text_control(
             self,
-            label=f'Loaded from: {json_file}',
+            label='Set values from another file',
             size=14)
         loading_box.Add(self.loaded_from)
-        loading_box.AddSpacer(10)
+        # loading_box.AddSpacer(5)
```

Comment:
feel free to remove if commented out code isn't needed

Reply:
Thanks, good catch.